packages/@react-spectrum/tabs/src/Tabs.tsx

```diff
@@ -270,6 +257,8 @@ export function TabList<T>(props: SpectrumTabListProps<T>) {
   }, [state.disabledKeys, state.selectedItem, state.selectedKey, props.children]);
   let stylePropsForVertical = orientation === 'vertical' ? styleProps : {};
 
+  let style : React.CSSProperties = collapse ? {visibility: 'hidden', position: 'absolute'} : {};
```

Comment:
This isn't quite right, if you go to `path=/story/tabs--collapse-behavior-isemphasized-true` and add a bunch of tabs, eventually, the screen will start scrolling. 

I think this would take care of it and allow the overflow calculation to still work
```suggestion
  let style : React.CSSProperties = collapse ? {maxWidth: 'calc(100% + 1px)', overflow: 'hidden', visibility: 'hidden', position: 'absolute'} : {maxWidth: 'calc(100% + 1px)'};
```

Reply:
Yes, that appears to work. I'm not sure where I missed that earlier when I was trying it out. Thanks for pointing it out.